backend/src/sms/routes/tests/sms-transactional.routes.test.ts

```diff
@@ -128,42 +127,6 @@ describe('POST /transactional/sms/send', () => {
     mockSendMessage.mockReset()
   })
 
-  test('Should return a HTTP 400 when template is invalid', async () => {
-    const mockSendMessage = jest
-      .spyOn(SmsService, 'sendMessage')
-      .mockRejectedValueOnce(new TemplateError('Invalid template'))
-    mockSecretsManager.getSecretValue().promise.mockResolvedValueOnce({
-      SecretString: JSON.stringify(TEST_TWILIO_CREDENTIALS),
-    })
-
-    const res = await request(app)
-      .post('/transactional/sms/send')
-      .set('Authorization', `Bearer ${apiKey}`)
-      .send(validApiCall)
-
-    expect(res.status).toBe(400)
-    expect(mockSendMessage).toBeCalledTimes(1)
-    mockSendMessage.mockReset()
-  })
-
-  test('Should return a HTTP 400 when recipient is not valid', async () => {
```

Comment:
invalid recipient error not caught at this point, handled separately, so this test is misleading

Reply:
Eh this is actually possible

Reply:
fixed [here](https://github.com/opengovsg/postmangovsg/compare/fix/twilio-error-2?expand=1), ps